pkg/plugin/aggregation/handler_test.go

```diff
@@ -232,3 +239,68 @@ func TestFilenameFromHeader(t *testing.T) {
 		})
 	}
 }
+
+func TestStatusFromRequest(t *testing.T) {
+	tcs := []struct {
+		desc        string
+		input       io.Reader
+		vars        map[string]string
+		expected    *plugin.StatusUpdate
+		expectedErr string
```

Comment:
Will more test cases get added here to check for error cases?

Reply:
Added one 👍